### PR TITLE
feat(pci.project): mix commander tracking of pci project creation

### PR DESCRIPTION
**Pull request by selm3n**
ref: TAPC-885

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #...TAPC-885 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/new/payment/controller.js [hunk 1]
```diff
@@ -361,6 +364,152 @@ export default class PciProjectNewPaymentCtrl {
...
+      )} Europe/Paris`;
+
+      window.tC.msr.additional_params += `&conversion_timestamp=${conversionTimestampString}`;
+
+      window.tC.msr.tc_rand = Math.random();
+      window.tC.msr.pixel = document.createElement('img');
+      window.tC.msr.pixel.style.display = 'none';
+      window.tC.msr.pixel.src = `https://${window.tC.msr.dns}/mix/o3/?tcs=${window.tC.msr.id_site}&idorder=${window.tC.msr.idorder}&amount=${window.tC.msr.amount}&currency=${window.tC.msr.currency}&user_id=${window.tC.msr.user_id}${window.tC.msr.additional_params}&rand=${window.tC.msr.tc_rand}`;
+      document.head.appendChild(window.tC.msr.pixel);
+    }
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/new/payment/controller.js:511]: **nitpick:** Can you extract the logic to load mix commander into a different file?

File: packages/manager/modules/pci/src/projects/new/payment/controller.js [hunk 2]
```diff
@@ -361,6 +364,152 @@ export default class PciProjectNewPaymentCtrl {
...
+      window.tC.msr.id_site = '3810';
+      window.tC.msr.idorder =
+        Date.now().toString(36) +
+        Math.random()
+          .toString(36)
+          .substring(2, 12)
+          .padStart(12, 0);
+      window.tC.msr.amount = '{{order_amount_tax_free}}';
+      window.tC.msr.currency = '{{currency code}}';
+      window.tC.msr.user_id = '{{customer ID}}';
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/new/payment/controller.js:470]: **nitpick:** The values in the single quotes must be dynamic values based on the logged in user and the total mount.

File: packages/manager/modules/pci/src/projects/new/payment/controller.js [hunk 3]
```diff
@@ -361,6 +364,152 @@ export default class PciProjectNewPaymentCtrl {
...
+        '{{payment method}}' +
+        '&discount_price=' +
+        '{{discount amount tax free}}';
+      window.tC.msr.additional_params +=
+        '&n_customer=' +
+        '{{true if the client is a new customer OR false if not}}' +
+        '&nc=' +
+        '{{true if the client is a new customer OR false if not}}' +
+        '&site_domain=' +
+        'www.ovh.com';
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/new/payment/controller.js:486]: **nitpick:** The values in the `{{}}` must be dynamic values.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/new/payment/controller.js [hunk 4]
```diff
@@ -361,6 +366,24 @@ export default class PciProjectNewPaymentCtrl {
   }
 
   trackOnPaymentFormSubmit(tag) {
+    if (this.$cookies.get(MANAGER_TRACKING) === '1') {
+      let isNewClient = true;
+      if (this.projects.length > 0) {
+        isNewClient = false;
+      }
+
```
Comment (MaximeBajeux) [inline packages/manager/modules/pci/src/projects/new/payment/controller.js:374]: I think you can just write
const isNewClient = this.projects.length === 0;

**Review by MaximeBajeux — CHANGES_REQUESTED** [review]

**Review by MaximeBajeux — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/new/payment/controller.js [hunk 5]
```diff
@@ -361,6 +366,21 @@ export default class PciProjectNewPaymentCtrl {
   }
 
   trackOnPaymentFormSubmit(tag) {
+    if (this.$cookies.get(MANAGER_TRACKING) === '1') {
```
Comment (ppprevost) [inline packages/manager/modules/pci/src/projects/new/payment/controller.js:369]: and if cookies is undefined ?

**Review by ppprevost — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/new/payment/controller.js (hunk 5 above)
Comment (selm3n) [reply packages/manager/modules/pci/src/projects/new/payment/controller.js:369]: Nothing happens. It's mentioned in the ticket to track the creation only if the user accepts the cookies policy

**Review by anooparveti — CHANGES_REQUESTED** [review]
Waiting the changes as discussed in DM.

**Pull request closed without merging**